finance-tracker/app/models/user.rb

```diff
@@ -5,4 +5,18 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  def stock_already_tracked?(ticker_symbol)
+    stock = Stock.check_db(ticker_symbol)
+    return false unless stock
+    stocks.where(id: stock.id).exists?
```

Comment:
```suggestion
    stocks.exists?(ticker: ticker_symbol)
```

orrect me if I am wrong but the condition if the stock is already tracked is if the stock is linked to the user record right?
if the above is true, then the above is enough